src/express/commands/setup-datadog.js

```diff
@@ -32,7 +32,7 @@ export async function setupDatadog() {
   let doc
   require('dotenv').config({ path: `${process.cwd()}/.env` })
 
-  if (process.env.TF_VAR_DOCKERIZED === 'yes') {
+  if (process.env.TF_VAR_DOCKERIZED === 'yes' || process.env.TF_VAR_PROJECT_ID) {
```

Comment:
Why is this in `||` with `TF_VAR_DOCKERIZED`?

Reply:
We did not test the datadog  setup for gcp. We will remove this once datadog testing is done. `TF_VAR_PROJECT_ID` is GCP mandatory var. So just added to current docker check.

Reply:
We have removed `TF_VAR_DOCKERIZED` and checking with `cloud === "gcp"`